modules/fundamental/src/product/model/mod.rs

```diff
@@ -26,3 +25,36 @@ impl ProductHead {
         })
     }
 }
+
+#[derive(Serialize, Deserialize, Debug, Clone, ToSchema)]
+pub struct ProductVersionHead {
+    pub id: Uuid,
+    pub version: String,
+    pub sbom_id: Option<Uuid>,
```

Comment:
We should add `#[serde(default, skip_serializing_if="Option::is_none")]`. Same for the others.

Reply:
Thanks, I missed that